source/js/apiHelpers.js

```diff
@@ -76,7 +84,7 @@ export function extractIDs(data) {
 export async function getRecipesByName(query, num = 5, offset = 0) {
   return new Promise((resolve, reject) => {
     const queryFormatted = query.trim().replace(/\s+/g, '-').toLowerCase();
-    fetch(`https://${HOST}/recipes/complexSearch?&query=${queryFormatted}&number=${num}&sort=popularity&offset=${offset}`, {
+    fetch(`https://${HOST}/recipes/complexSearch?query=${queryFormatted}&number=${num}&sort=popularity&offset=${offset}`, {
```

Comment:
The old one with `?&query` does works for me. Getting rid of `{}` from `{ API_KEY } ` should fix the no result problem when using the API key directly.